### DISCUSSION TITLE
Number of Repositories

### DISCUSSION THREAD

**Discussion by Abdullah5111** [community/community#114646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I hope you are doing well.
I discovered an issue on the frontend side. The number of repositories before and after deleting a repo remains the same until I reload the page. 
![GithubFix1](https://github.com/community/community/assets/122647095/3f38ba80-d4fd-484f-b00d-4926a7afd960)
![GithubFix2](https://github.com/community/community/assets/122647095/ad2bc8f0-1382-48ad-b472-dbdd5a361908)

**Comment by maksymhu** [comment]
If you delete repository, github page didn't automatically load?

**Comment by amardotai** [comment]
Github doesn't use states in few of it's frontend elements so you need to reload the page automatically in order to confirm any changes.
